_docs/developer/getting_started/commit_to_PR_from_fork.md

```diff
@@ -22,49 +22,52 @@ The instructions below are for command line use of git.
 
 
 1.  First, from the PR on the Github website, find the name of the
-    fork + branch name.  It should be formatted something like this:
+    user + branch name.  It should be formatted something like this:
     ```
     contributorusername:contributor_branch_name
     ```
 
+    Additionally, find the name of the fork
+    on the fork's page on Github.
+    
 
-2.  Next, make a local branch on your computer in your working repository
-    with the proposed code changes.  Here are the command lines
-    (substitute the fork & branch name we found above):
+3.  Next, make a local branch on your computer in your working repository
```

Comment:
why did you change this number to 3? 
Did you mean to insert a step 2 above?

Reply:
Sorry about that; the file already had issues with the numbering which i passed through and fixed now